fabric-entity-attribute-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-entity-attribute-api-v1",
+  "name": "Fabric Entity Attribute API (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-entity-attribute-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.8.0",
+    "minecraft": ">=1.16-alpha.20.15.a"
+  },
+  "description": "Hooks for registering .",
```

Comment:
Maybe complete this description message?

Reply:
Yeah guess I forgot to update this

Reply:
Done, thanks